src/commandLine.js

```diff
@@ -115,12 +118,34 @@ var CommandLineManager = {
       }
     }
 
+    // process proxy host
+    index = argv.indexOf('--scheme');
+    if (index !== -1) {
+      if (CommandLineManager.validArgValue(argv[index + 1])) {
+        var scheme = argv[index + 1];
+        if (!(scheme == 'http' || scheme == 'https')){
+          console.log("\nScheme can only be http/https");
+          invalidArgs.add('--scheme');
+        }
+        else{
+          RdGlobalConfig.SCHEME = scheme;
+          argv.splice(index, 2);  
+        }
```

Comment:
```suggestion
        } else {
          RdGlobalConfig.SCHEME = scheme;
          argv.splice(index, 2);  
        }
```
for conventions... you can refer the following: https://github.com/airbnb/javascript#table-of-contents (non-ES5)...

Reply:
can be resolved